website/source/docs/providers/chef/r/data_bag.html.markdown

```diff
@@ -6,7 +6,7 @@ description: |-
   Creates and manages a data bag in Chef Server.
 ---
 
-# chef\_data\_bag
+# chef_data_bag
```

Comment:
Fair enough... I thought I remembered trying that once and seeing it italic it, but probably what happened  instead was that I saw the escapes and thought "oh, that must be because it italicizes them" and then never actually questioned it. So okay! Looking forward to not having to insert backslashes anymore.